src/main/java/Duke.java

```diff
@@ -1,10 +1,409 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * A decorative prefix added to the beginning of lines.
+     */
+    public static final String LINE_PREFIX = "\t ";
+
+    public static final String HORIZONTAL_LINE = LINE_PREFIX
+            + "____________________________________________________________";
+
+    /**
+     * A platform independent line separator.
+     */
+    public static final String LS = System.lineSeparator() + LINE_PREFIX;
+
+    public static final String COMMAND_LIST_WORD = "list";          // Keyword for listing tasks
+    public static final String COMMAND_BYE_WORD = "bye";            // Keyword for exiting program
+    public static final String COMMAND_ADD_WORD = "add";            // Keyword for adding a new task
+    public static final String COMMAND_DONE_WORD = "done";          // Keyword for marking a task as done
+    public static final String COMMAND_TODO_WORD = "todo";          // Keyword for adding a new todo
+    public static final String COMMAND_DEADLINE_WORD = "deadline";  // Keyword for adding a new deadline
+    public static final String COMMAND_EVENT_WORD = "event";        // Keyword for adding a new event
+
+    public static final String MESSAGE_LIST_TITLE = "Here are the tasks in your list:";
+    public static final String MESSAGE_DONE_TITLE = "Nice! I've marked this task as done:";
+    public static final String MESSAGE_ADD_TITLE = "Got it. I've added this task:";
+    public static final String MESSAGE_ADD_CONCLUSION = "Now you have %d tasks in the list.";
+    public static final String MESSAGE_FOR_INVALID_INPUT = "Invalid command.";
+
```

Comment:
Constants used for most outputs in the command line, easy readability

Reply:
Thank you very much for your comments.